pkg/core/localize/locales/en/cmd/registry_crud.en.toml

```diff
@@ -211,6 +211,17 @@ one = 'Filtering Service Registry instances with the query "{{.Search}}".'
 description = 'Info message when no Registry instances were found'
 one = 'No Service Registry instances were found.'
 
+[registry.common.log.message.compatibleAPIs]
+one='''
+You can connect to the Service Registry instance using following options:
+ - For native Apicurio API and SerDes - Core Registry API:
+    {{.CoreRegistryAPI}}
+ - For any upstream tools - Confluent Schema Registry API:  
+    {{.SchemaRegistryAPI}}
+ - For tools compatible with CNCF API - CNCF Schema Registry API
```

Comment:
Printing this using `logger.Info()`. 
Might need a better description for CNCF endpoint.
Cc @wtrocki

Reply:
Actually that was the only one that were right :)